pkg/api/rest/rest.go

```diff
@@ -125,6 +125,13 @@ func (w GracefulDeleteAdapter) Delete(ctx api.Context, name string, options *api
 	return w.Deleter.Delete(ctx, name)
 }
 
+// CollectionDeleter is an object that can delete a collection
+// of RESTful resources.
+type CollectionDeleter interface {
+	// TODO: Add comment.
```

Comment:
Fix the // TODO and actually add the comment on how this is used, and when to use it.